auslib/test/test_db.py

```diff
@@ -2752,20 +2756,53 @@ def testOnUpdate(self):
         def doit():
             self.db.rules.update({"rule_id": 2}, {"product": "blah"}, "bob", 1)
         mock_conn = self._runTest(doit)
+        # Updating a Rule causes its Scheduled Change to be updated as well, so we have to check both of those calls.
+        mock_conn.sendmail.assert_any_call("fake@from.com", "fake@to.com", PartialString("UPDATE"))
+        mock_conn.sendmail.assert_any_call("fake@from.com", "fake@to.com", PartialString("Row(s) to be updated as follows:"))
+        mock_conn.sendmail.assert_any_call("fake@from.com", "fake@to.com", PartialString("'product': None ---> 'blah'"))
+        mock_conn.sendmail.assert_any_call("fake@from.com", "fake@to.com", PartialString("'channel': u'release' (unchanged)"))
         mock_conn.sendmail.assert_called_with("fake@from.com", "fake@to.com", PartialString("UPDATE"))
```

Comment:
Where we are modifying two tables it would be good to check for the table names, eg "UPDATE to rules" for the first one.